### PR TITLE
New Asciidoc Cheat Sheet

### PR DESCRIPTION
**Pull request by phora**
---
https://duck.co/ia/view/asciidoc_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@phora Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/asciidoc.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
+{
+    "id": "asciidoc_cheat_sheet",
+    "name": "Asciidoc",
+    "description": "A text document format for writing notes, documentation, articles, books, ebooks, slideshows, web pages, man pages and blogs",
+    "metadata": {
+		"sourceName": "Asciidoc cheat sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/asciidoc.json]: Typically we use the domain name as the source, to indicate where the user is going. That obviously works best when the domain is something others might recognize, so I think what you have is OK.

I would capitalize "Cheat Sheet" though

File: share/goodie/cheat_sheets/json/asciidoc.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+		"sourceUrl": "http://powerman.name/doc/asciidoc"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/asciidoc.json]: It looks like you're using 8-space hard tabs? Can you please switch to 4-space soft tabs?

Comment (phora) [inline share/goodie/cheat_sheets/json/asciidoc.json]: Fixing ASAP. Codio normally inserts 4-space soft tabs, but the hard tabs must've come from copypasting the metadata portion.

File: share/goodie/cheat_sheets/json/asciidoc.json [hunk 3]
```diff
@@ -0,0 +1,167 @@
+{
+    "id": "asciidoc_cheat_sheet",
+    "name": "Asciidoc",
+    "description": "A text document format for writing notes, documentation, articles, books, ebooks, slideshows, web pages, man pages and blogs",
+    "metadata": {
+        "sourceName": "Asciidoc Cheat Sheet",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/asciidoc.json]: i think sourceName should be: powerman

File: share/goodie/cheat_sheets/json/asciidoc.json [hunk 4]
```diff
@@ -0,0 +1,167 @@
+{
+    "id": "asciidoc_cheat_sheet",
+    "name": "Asciidoc",
+    "description": "A text document format for writing notes, documentation, articles, books, ebooks, slideshows, web pages, man pages and blogs",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/asciidoc.json]: please make description a little shorter

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=asciidoc cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/asciidoc_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@phora this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

If you need ideas about how to contribute next, please feel free to reach out to me on Slack! I'm happy to chat about how you can help :+1:

File: share/goodie/cheat_sheets/json/asciidoc.json [hunk 5]
```diff
@@ -0,0 +1,167 @@
+{
+    "id": "asciidoc_cheat_sheet",
+    "name": "Asciidoc",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/asciidoc.json]: nit: It looks like at the source they capitalize the "D" in AsciiDoc

**Pull request merged**